extensions/amp-analytics/0.1/test/test-vendors.js

```diff
@@ -14,12 +14,37 @@
  * limitations under the License.
  */
 
-import {ANALYTICS_CONFIG} from '../vendors';
+import {ANALYTICS_CONFIG, ANALYTICS_IFRAME_TRANSPORT_CONFIG} from '../vendors';
 
-describe('analyaitcs vendors', () => {
+describe('analytics vendors', () => {
   it('googleanalytics & googleanalytics-alpha should be identical', () => {
     const gaConfig = ANALYTICS_CONFIG['googleanalytics'];
     expect(gaConfig).to
         .deep.equal(ANALYTICS_CONFIG['googleanalytics-alpha']);
   });
+
+  it('should contain only iframe transport in ' +
+      'ANALYTICS_IFRAME_TRANSPORT_CONFIG', () => {
+    for (const vendor in ANALYTICS_IFRAME_TRANSPORT_CONFIG) {
+      const vendorITEntry = ANALYTICS_IFRAME_TRANSPORT_CONFIG[vendor];
+      expect(Object.keys(vendorITEntry).length).to.equal(1);
+      expect(vendorITEntry.transport).to.exist;
+      expect(Object.keys(vendorITEntry.transport).length).to.equal(1);
+      expect(vendorITEntry.transport.iframe).to.exist;
+    }
+  });
+
+  it('Should not contain iframe transport in ANALYTICS_CONFIG (other than' +
+      ' those in ANALYTICS_IFRAME_TRANSPORT_CONFIG)', () => {
+    for (const vendor in ANALYTICS_CONFIG) {
+      const vendorEntry = ANALYTICS_CONFIG[vendor];
+      if (vendorEntry.hasOwnProperty('transport') &&
+          vendorEntry.transport.hasOwnProperty('iframe')) {
+        const vendorITEntry = ANALYTICS_IFRAME_TRANSPORT_CONFIG[vendor];
+        expect(vendorITEntry).to.exist;
+        expect(vendorITEntry.transport).to.exist;
```

Comment:
Are these necessary?  Aren't they covered by the previous test?  Or you could combine into one test...

Reply:
Agreed - lines 45,46 are duplicating the previous test. Removing.
I would prefer to not combine the two tests together. In the one case, we're looping over ANALYTICS_IFRAME_TRANSPORT_CONFIG, and in the other case over ANALYTICS_CONFIG. Also, if the more expect(something).to.exist calls we have in a single test, the harder it is to figure out the culprit when one of them fails, since Travis doesn't give consistent line numbers in that case.

Reply:
Done